pkgs/development/libraries/libevent/default.nix

```diff
@@ -12,9 +12,10 @@ stdenv.mkDerivation {
   };
 
   nativeBuildInputs = [ autoreconfHook ];
-  buildInputs = [ python ];
 
-  patchPhase = ''
+  buildInputs = stdenv.lib.optional (python ? isPy2) python;
```

Comment:
Thing it, it only works with python2, so I figure why even include it in the build inputs if the python we're using isn't python2.